librz/core/cmd_descs_generate.py

```diff
@@ -59,7 +59,11 @@
 DESC_HELP_ARG_TEMPLATE_OPTIONAL = '.optional = {optional}, '
 DESC_HELP_ARG_TEMPLATE_NO_SPACE = '.no_space = {no_space}, '
 DESC_HELP_ARG_TEMPLATE_DEFAULT_VALUE = '.default_value = {default_value}, '
```

Comment:
put a `\t\t` at the beginning of all these (DESC_HELP_ARG_TEMPLATE_DEFAULT_VALUE, DESC_HELP_ARG_TEMPLATE_NO_SPACE, DESC_HELP_ARG_TEMPLATE_OPTIONAL, etc.)

Reply:
It enlarges the difference. I tried.